<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lqPlayerControlsToolbar</class>
 <widget class="QToolBar" name="lqPlayerControlsToolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>39</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>VCR Controls</string>
  </property>
  <property name="orientation">
   <enum>Qt::Horizontal</enum>
  </property>
  <property name="PV_MUST_BE_MASTER" stdset="0">
   <bool>true</bool>
  </property>
  <action name="actionVCRPlay">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="lqResources.qrc">
     <normaloff>:/lqResources/Icons/media-playback-start.png</normaloff>:/lqResources/Icons/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Play</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionVCRPreviousFrame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="lqResources.qrc">
     <normaloff>:/lqResources/Icons/media-seek-backward.png</normaloff>:/lqResources/Icons/media-seek-backward.png</iconset>
   </property>
   <property name="text">
    <string>Pre&amp;vious Frame</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
   <property name="PV_MUST_BE_MASTER" stdset="0">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionVCRFirstFrame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="lqResources.qrc">
     <normaloff>:/lqResources/Icons/media-skip-backward.png</normaloff>:/lqResources/Icons/media-skip-backward.png</iconset>
   </property>
   <property name="text">
    <string>&amp;First Frame</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="actionVCRNextFrame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="lqResources.qrc">
     <normaloff>:/lqResources/Icons/media-seek-forward.png</normaloff>:/lqResources/Icons/media-seek-forward.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Next Frame</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionVCRLastFrame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="lqResources.qrc">
     <normaloff>:/lqResources/Icons/media-skip-forward.png</normaloff>:/lqResources/Icons/media-skip-forward.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Last Frame</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionVCRLoop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="lqResources.qrc">
     <normaloff>:/lqResources/Icons/media-loop.png</normaloff>:/lqResources/Icons/media-loop.png</iconset>
   </property>
   <property name="text">
    <string>L&amp;oop</string>
   </property>
   <property name="iconText">
    <string>Loop</string>
   </property>
   <property name="toolTip">
    <string>Loop</string>
   </property>
   <property name="statusTip">
    <string>Loop</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="icon">
    <iconset resource="lqResources.qrc">
     <normaloff>:/lqResources/Icons/lqRecord.png</normaloff>:/lqResources/Icons/lqRecord.png</iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <addaction name="actionVCRFirstFrame"/>
  <addaction name="actionVCRPreviousFrame"/>
  <addaction name="actionVCRPlay"/>
  <addaction name="actionVCRNextFrame"/>
  <addaction name="actionVCRLastFrame"/>
  <addaction name="actionVCRLoop"/>
  <addaction name="actionRecord"/>
 </widget>
 <resources>
  <include location="lqResources.qrc"/>
 </resources>
 <connections/>
</ui>
